src/src.builds

```diff
@@ -32,4 +33,18 @@
                       DepsExceptions="@(ExceptionForDepsJson)"
                       OutputPath="$(_OutputTestSharedFrameworkDepsPath)"/>
   </Target>
+
+  <Target Name="GenerateXUnitRunnerConfigFile" AfterTargets="BuildAllProjects" Condition="'$(TargetGroup)' == 'netfx'">
```

Comment:
Is TargetGroup netfx correct here? Should it be _bc_TargetGroup? (perhaps they are always the same for builds files).